### DISCUSSION TITLE
Will there be new updates in the future?

### DISCUSSION THREAD

**Discussion by BeastlordGG** [hythloday/VenturePlanSoDMissions#68] [Q&A] [answered]
now comes 9.2 and a couple of new missions they will be added to ventureplanSoDMissions?
I think you do a good job and I appreciate them

Regards Beastlord

**Comment by zealvurte** [comment, marked as answer]
Unlikely, as @hythloday has been absent for a long time, but others will probably still do pull requests, and I've now posted [the new spell data for 9.2](https://github.com/hythloday/VenturePlanSoDMissions/issues/69).
